packages/client/src/types.ts

```diff
@@ -91,8 +118,19 @@ export type Topic = OloTopic | IMTRTopic | RedirectToOloTopic;
 /**
  * Checker related types
  */
-export type BooleanOption = {
+export type AnswerOptions = {
```

Comment:
Why plural? Isn't this one AnswerOption?

Reply:
Because I was thinking: one answer have multiple options: formValue, label and value... I don't like this as well. The name AnswerOption(s) seem to be a combination of Answer (from frontend) and options (from IMTR).

Let's just make this `Answer`. What do you think?

Reply:
Plain `Answer` makes a lot more sense and importing `Answer` from `imtr-client` will be imported like `{ Answer as IMTRAnswer }`

Reply:
I've added this situation to the dev-meeting topics